types/jquery/misc.d.ts

```diff
@@ -4712,10 +4708,6 @@ $( "ul" ).click( handler ).find( "ul" ).hide();
         target: TTarget;
     }
 
-    interface EventLike {
```

Comment:
Why were these added in the first place? Seems like 869c4ba5605d0dcdbe7804edae160dd2f984f1e3 introduced `EventLike`

Reply:
`EventLike` was used by an `Event` constructor overload. The overload does work but is not documented. I believe it's meant to be internal.